hack/generator/pkg/astmodel/object_type.go

```diff
@@ -526,3 +527,10 @@ func extractEmbeddedTypeName(t Type) (TypeName, error) {
 
 	return TypeName{}, errors.Errorf("embedded property type must be TypeName, was: %T", t)
 }
+
+// DebugDescription adds a description of the current type to the passed builder
+// builder receives the full description, including nested types
+// types is a dictionary for resolving named types
+func (objectType *ObjectType) WriteDebugDescription(builder *strings.Builder, types Types) {
+	builder.WriteString("Object")
```

Comment:
Technically could print the properties and stuff here?

Reply:
I considered it, but was hesitant about flooding the output with a LOT more content.